vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -84,6 +84,20 @@ elif [ "${OS_TYPE}" = "Linux" ] && grep -q "cvm" <<< "$FEATURE_FLAGS"; then
       } \
     } \
   }"
+elif [ "${OS_TYPE}" = "Linux" ] && grep -q "GB200" <<< "$FEATURE_FLAGS"; then
```

Comment:
I guess we're not using this feature flag to do anything specific at build time (yet)?